### ISSUE TITLE
Contribution welcome in following topics

### ISSUE THREAD

**Issue by blackbone** [blackbone/other-ecs-benchmarks#1]
# Verification

## Ecs implementations

Require help to verify ECS utilization implemented in proper way:
- [ ] Arch
- [ ] Fennecs
- [ ] Morpeh

## Behncmarks

Require feedback about Context API in following topics:
- Setup
- Warmup
- Lock \ Commit
- CRUD

# Implementation

## Ecs

All other Ecs framework contexts implementations are very welcome

## Benchmarks

Need some ideas about use cases which will affect following topics:
- Structural Changes
- Long runs
- Complex Querying

Cheers!

**Comment by yuliuren** [comment]
https://github.com/DesertImage/ecs-burst Maybe you can add this to the list

**Comment by blackbone** [comment]
@yuliuren it's not Unity benchmarks but if there's no unity dependencies and you'll add a csproj so it will work in pure dotnet - it's possible
but as far as i can see it has deps to Unity.

**Comment by andreakarasho** [comment]
Maybe you are interested to add this too https://github.com/andreakarasho/TinyEcs
Thank you!

**Comment by blackbone** [comment]
@andreakarasho yep, it's planned but have not much time for it right now, will add later or you can add it via PR.

**Comment by MiKiNuo** [comment]
Maybe you are interested to add this too https://github.com/sebas77/Svelto.ECS
Thank you!

**Comment by blackbone** [comment]
@MiKiNuo may be a bit later, or you can try do do it yourself. Feel free to create PR.
For now planning to add some more complex scenarios.
